docs/index.rst

```diff
@@ -5,9 +5,10 @@
     :caption: Table of Contents:
     :hidden:
 
-    generic/theory
     generic/install
     python/index
+    python/tutorial
```

Comment:
python/tutorial doesn't exist i.e. this will fail. Currently the tutorials can already be accessed through python/index which can serve as a top-level for the Python-specific documentation